.github/workflows/main.yml

```diff
@@ -21,10 +21,10 @@ jobs:
           arch=$(go env GOARCH)
 
           # download kubebuilder and extract it to tmp
-          curl -L https://go.kubebuilder.io/dl/2.3.1/${os}/${arch} | tar -xz -C /tmp/
+          curl -L https://go.kubebuilder.io/dl/3.3.0/${os}/${arch} | tar -xz -C /tmp/
 
           # move to a long-term location and put it on your path
-          sudo mv /tmp/kubebuilder_2.3.1_${os}_${arch} /usr/local/kubebuilder
+          sudo mv /tmp/kubebuilder_3.3.0_${os}_${arch} /usr/local/kubebuilder
```

Comment:
Hmm this is interesting. I was under the impression that with kubebuilder 3 you don't have to install the kubebuilder binaries anymore because they come through the makefile (it installs the api-server and etcd binary)

Reply:
Ahh I've missed it. Will eliminate these with downloads